src/test/java/seedu/anilist/logic/parser/FindCommandParserTest.java

```diff
@@ -9,6 +9,8 @@
 
 import seedu.anilist.logic.commands.FindCommand;
 import seedu.anilist.logic.parser.exceptions.ParseException;
+import seedu.anilist.model.anime.Name;
+import seedu.anilist.model.genre.Genre;
```

Comment:
If we are just using the messages would it be better to just import the messages themselves rather than the class?

Reply:
Yes, will change that.

Reply:
oh right i didnt realize that they had the same name..